Backend/src/config/keys.ts

```diff
@@ -0,0 +1,26 @@
+// types
+interface GoogleKeys {
+  clientID: string;
+  clientSecret: string;
+}
+interface sessionKey {
+  cookieKey: string;
+}
+
+interface Keys {
+  google: GoogleKeys;
+  session: sessionKey;
+}

```

Comment:
Lets name AppConfig

and add GoogleAuthConfig
and SessionConfig.

Something also good to do (or I can do it later) is to read these secrets from .env file.
but you don't need to worry about that at all for now. you can just keep in this file at the moment.